full-service/src/service/wallet.rs

```diff
@@ -36,6 +117,6 @@ pub fn rocket(
     state: WalletState<ThickClient<HardcodedCredentialsProvider>, FogResolver>,
 ) -> rocket::Rocket {
     rocket::custom(rocket_config)
-        .mount("/", routes![wallet_api, wallet_help])
+        .mount("/", routes![wallet_api, wallet_help_v2, wallet_help_v1])
```

Comment:
I'm not sure what the convention is for JSON-RPC, but I have often seen versioned REST APIs use a URL scheme like /api/v1/foo, /api/v2/foo. I think that more clearly conveys the hierarchy than having things like /wallet --> v2, and /wallet/v1 --> v1.

Maybe something like this would be nice:

```
.mount("/api/v1", routes![wallet_api_v1, wallet_help_v1])
.mount("/api/v2", routes![wallet_api_v2, wallet_help_v2])
```

Reply:
We discussed offline and decided we will not be maintaining v1 in a backward-compatible way. In the future though, this versioning scheme may be what we decide to do.